app/views/articles/edit.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1>Edit Article</h1>
+<!-- Page Header -->
+<header class="masthead" style="background-image: url(<%= asset_path 'create-bg.jpg' %>);">
```

Comment:
You are using a `header` in several pages for the articles. How can you refactor the code to avoid this repetition?

Reply:
With `yield` and `content_for` I refactor the `header` and modify the necessary text in each view.